ironfish/src/rpc/routes/chain/getAsset.ts

```diff
@@ -7,22 +7,14 @@ import { Assert } from '../../../assert'
 import { FullNode } from '../../../node'
 import { CurrencyUtils } from '../../../utils'
 import { NotFoundError, ValidationError } from '../../adapters'
+import { RpcAsset } from '../../types'
 import { ApiNamespace, routes } from '../router'
 
 export type GetAssetRequest = {
   id: string
 }
 
-export type GetAssetResponse = {
-  createdTransactionHash: string
-  id: string
-  metadata: string
-  name: string
-  nonce: number
-  creator: string
-  owner: string
-  supply: string
-}
+export type GetAssetResponse = Omit<RpcAsset, 'status'>
```

Comment:
I believe that the chain doesn't have access to the status field, but lmk if I'm wrong. If that's the case, it would simplify everything to 1 asset model. Alternatively, I can return status unknown, but I figured it would be more confusing for the users.

Reply:
`status` is computed based on the sequence of the block that includes the transaction that created the asset

I think we can compute it in this endpoint to simplify to a single asset model